Assets/TemporaryCopyBones.cs

```diff
@@ -0,0 +1,15 @@
+﻿using UnityEngine;
+
+namespace SS3D
+{
+    public class TemporaryCopyBones : MonoBehaviour
```

Comment:
What is the purpose of this class? Maybe you could add a comment.

Reply:
Deleted the class